src/rootfs/files/scripts/cleanup_rootfs.sh

```
@@ -5,42 +5,77 @@ set -euo pipefail;
+        dpkg -r \
             build-essential \
             g++-13 \
             g++-13-x86-64-linux-gnu \
             g++-x86-64-linux-gnu \
-            linux-headers-6.12.13-nvidia-gpu-confidential \
             g++ \
-            gcc';
-    rm -rf ${OUTPUTDIR}/tmp/*;
-    rm -rf ${OUTPUTDIR}/usr/share/{bash-completion,bug,doc,info,lintian,locale,man,menu,misc,pixmaps,zsh};
+            gcc \
+            $kernel_headers';
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)